src/pages/HomePage/hooks.ts

```diff
@@ -0,0 +1,40 @@
+import { useMemo } from "react";
+import { get } from "lodash-es";
+import {
+  useQueryWithClient,
+  useDeskproLatestAppContext,
+} from "@deskpro/app-sdk";
+import { getEntityListService } from "../../services/deskpro";
+import { getDevicesService } from "../../services/lansweeper";
+import { enhanceDevices } from "../../utils";
+import { QueryKey } from "../../query";
+import type { DPUser, DeviceType, UserContext } from "../../types";
+
+type UseLinkedDevices = () => {
+  isLoading: boolean;
+  devices: DeviceType[];
+};
+
+const useLinkedDevices: UseLinkedDevices = () => {
+  const { context } = useDeskproLatestAppContext() as { context: UserContext };
+  const dpUserId = get(context, ["data", "user", "id"]);
```

Comment:
Replace `get` with JS/TS native `foo?.bar`- no need for lodash.

Reply:
done